test/extended/openstack/local-bd-etcd.go

```diff
@@ -102,15 +78,20 @@ var _ = g.Describe("[sig-installer][Suite:openshift/openstack] The OpenShift clu
 		o.Expect(err).NotTo(o.HaveOccurred())
 
 		skipUnlessMachineAPIOperator(ctx, dc, clientSet.CoreV1().Namespaces())
-		skipUnlessEtcdAdditionalBlockDevice(ctx, dc)
 
+		cpms, err = getControlPlaneMachineSet(ctx, dc)
+		if err != nil {
+			e2eskipper.Skipf("Failed to get control plane machine sets: %v", err)
+		}
+
+		skipUnlessEtcdAdditionalBlockDevice(cpms)
 	})
 
 	g.It("runs with etcd on ephemeral local block device", func() {
 
 		g.By("checking that the control plane flavor has enough ephemeral storage")
 		{
-			controlPlaneFlavor = clusterCPMS.Get("spec.template.spec.providerSpec.value.flavor").String()
+			controlPlaneFlavor = cpms.Get("spec.template.machines_v1beta1_machine_openshift_io.spec.providerSpec.value.flavor").String()
```

Comment:
You may want to declare  the provider spec value spec.template.machines_v1beta1_machine_openshift_io.spec.providerSpec.value once

Reply:
done